bookstore/client/notebook.py

```diff
@@ -182,12 +204,11 @@ def __init__(self):
         self.nb_client = NotebookClientCollection.current_server()
         super().__init__(self.nb_client.nb_config)
         self.session = self.sessions[self.kernel_id]
-        self.notebook = NotebookSession(**self.session).notebook
 
     @property
     def connection_file(self):
         return get_ipython().parent.parent.connection_file
 
     @property
     def kernel_id(self):
-        return os.path.basename(self.connection_file).lstrip('kernel-').rstrip('.json')
+        return extract_kernel_id(self.connection_file)
```

Comment:
Do we anticipate that this function (extract_kernel_id) will be used more than once? If not, it may be better to leave as is.